app/grandchallenge/teams/urls.py

```diff
@@ -13,7 +13,7 @@
 app_name = "teams"
 
 urlpatterns = [
-    path("", TeamList.as_view(), name="list"),
+    path("all/", TeamList.as_view(), name="list"),
```

Comment:
Maybe, I think otherwise there would be a clash with `all` as a slug.

```suggestion
    path("t/all/", TeamList.as_view(), name="list"),
```

Reply:
Agreed!